example/src/components/RoomParticipant.tsx

```diff
@@ -0,0 +1,100 @@
+import { BrandColors, AdditionalColors, TextColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { Icon } from './Icon';
+import { NoCameraView } from './NoCameraView';
+import { Typo } from './Typo';
+
+type RoomParticipantProps = {
+  tracks: Membrane.Track[];
+  metadata: Membrane.Metadata;
+  type: Membrane.ParticipantType;
+};
+
+export const RoomParticipant = ({
+  tracks,
+  metadata,
+  type,
+}: RoomParticipantProps) => {
```

Comment:
you can pass the whole participant and destructure it here, something like that (haven't tested it):
```
export const RoomParticipant = ({{participant: {tracks, metadata, type}}) => {
...
}
```
if I add something to the participant object later or you'll want to change `RoomParticipant` component that would be less changes.

Reply:
Done.